godoc/static/godoc.html

```diff
@@ -33,11 +33,12 @@
 </div><!-- #lowframe -->
 
 <div id="topbar"{{if .Title}} class="wide"{{end}}><div class="container">
-<div class="top-heading" id="heading-wide"><a href="/pkg/">Go Documentation Server</a></div>
```

Comment:
`/pkg/` is not necessary in any case because `/` is already redirected by handler